### PR TITLE
CLDR-15662 LocaleCompletion (third) Meter: use baselineCount

### PR DESCRIPTION
**Pull request by btangmu**
-Change how the front-end 3rd meter calculates percentage, using baselineCount

-Percentage cannot be less than 0 or greater than 100

CLDR-15662

- [ ] This PR completes the ticket.

### CODE REVIEW

**Comment by btangmu** [comment]
this changes the 3rd meter

it doesn't change Priority Items Summary "Progress" column yet -- that will be next

File: tools/cldr-apps/js/src/esm/cldrProgress.js [hunk 1]
```diff
@@ -188,12 +188,21 @@ function refreshVoterMeter() {
...
+      currentProblems >= baselineProblems
+        ? 0
+        : baselineProblems - currentProblems;
+    progressWrapper.updateLocaleMeter(
       new MeterData(
         cldrText.get("progress_all_vetters"),
-        localeProgressStats.votes,
-        localeProgressStats.total,
+        solvedProblems,
+        baselineProblems,
```
Comment (btangmu) [inline tools/cldr-apps/js/src/esm/cldrProgress.js:205]: I'm thinking we should switch to a better terminology for all 3 meters: "done" and "total".

Here solvedProblems would be "done", and baselineProblems would be "total".

For voter completion and page completion, votes would be "done" and total would be "total".

Comment (macchiati) [reply tools/cldr-apps/js/src/esm/cldrProgress.js:205]: sounds good.

**Review by macchiati — COMMENTED** [review]
Looks good, but I suggest that we don't integrate until @srl295 has a chance to look at the baseline issue (see email).

**Comment by btangmu** [comment]
See https://github.com/unicode-org/cldr/pull/2115 which combines these 3rd-meter changes with changes for Priority Items Summary

**Pull request closed without merging**